Makefile

```diff
@@ -52,56 +54,67 @@ test:
 test-sanity:
 	go test -v -cover ./tests/sanity/... -coverprofile=cov2.out
 
+# build nightly image
 .PHONY: image-nightly
 image-nightly:
 	# Build image with newest juicefs-csi-driver and juicefs
-	docker build --build-arg TARGETARCH=$(TARGETARCH) -t $(IMAGE):nightly .
-
-.PHONY: push
-push-nightly:
-	docker tag $(IMAGE):nightly $(REGISTRY)/$(IMAGE):nightly
-	docker push $(REGISTRY)/$(IMAGE):nightly
+	docker build --build-arg TARGETARCH=$(TARGETARCH) \
+		--build-arg JUICEFS_MOUNT_IMAGE=juicedata/mount:nightly-$(JUICEFS_EE_LATEST_VERSION) \
```

Comment:
I think we should add Makefile env `JUICEFS_MOUNT_IMAGE` to modify it in make time.